broken.js

```diff
@@ -0,0 +1,3 @@
+var hello = function() {
```

Comment:
_[farcy v1.0.rc1](https://github.com/appfolio/farcy)_
* "hello" is defined but never used (no-unused-vars)
* Missing function expression name. (func-names)
* Unexpected var, use let or const instead. (no-var)